code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -127,6 +127,30 @@
 	mag_display = TRUE
 	empty_indicator = TRUE
 
+/obj/item/gun/ballistic/automatic/wt650
+	name = "WT-650 Assault Rifle"
+	desc = "A long-barreled, assault variant of the WT-550. Uses 5.56 rounds and is specially designed for assault teams, PMC forces, and armies favouring balistic weapons over their laser counterparts."
+	icon_state = "wt650"
+	item_state = "wt650"
+	mag_type = /obj/item/ammo_box/magazine/wt650
+	fire_delay = 2
+	burst_size = 1
+	can_suppress = FALSE
+	can_bayonet = TRUE
+	knife_x_offset = 27
+	knife_y_offset = 12
+	mag_display = TRUE
+	mag_display_ammo = TRUE
+	weapon_weight = WEAPON_HEAVY
+
+/obj/item/gun/ballistic/automatic/wt650/peacekeeper // So peacekeepers can start with a rubber mag instead of a lethal one.
+	mag_type = /obj/item/ammo_box/magazine/wt650/rubber
+	weapon_weight = WEAPON_HEAVY
+
+/obj/item/gun/ballistic/automatic/wt650/amber // So ATF members can have laser ammo because idk its cool.
+	mag_type = /obj/item/ammo_box/magazine/wt650/laser
+	weapon_weight = WEAPON_HEAVY
```

Comment:
```suggestion
```

Reply:
Without it there, same with the other, weapons were not having a weight set.